content/en/hosting/4.x/production/docker/_index.md

```diff
@@ -1,17 +1,160 @@
+87415a2d91ea   public.ecr.aws/s5s3h4s7/cht-haproxy:4.0.1-4.0.1               "/entrypoint.sh -f /…"    17 minutes ago   Up 8 minutes   5984/tcp                                                                   cht_haproxy_1
+58454457467a   public.ecr.aws/s5s3h4s7/cht-couchdb:4.0.1-4.0.1               "tini -- /docker-ent…"    17 minutes ago   Up 8 minutes   4369/tcp, 5984/tcp, 9100/tcp                                               cht_couchdb_1
+d01343658f3f   public.ecr.aws/s5s3h4s7/cht-upgrade-service:latest            "node /app/src/index…"    17 minutes ago   Up 8 minutes                                                                              upgrade-service-cht-upgrade-service-1
+```
+
+This should show related to the CHT core are running
+* cht_nginx
+* cht_api
+* cht_sentinel
+* cht_couchdb
+* cht_healthcheck
+* cht_haproxy
+* cht-upgrade-service
+
+Take note of the `STATUS` column and make sure no errors are displayed there. If any container is restarting or mentioning any other error, check the logs using the `docker logs <container-name>` command.
+
+If all has gone well, nginx should now be listening at both port 80 and port 443. Port 80 has a permanent redirect to port 443, so you can only access the CHT using https.
+
+To login as the `medic` user in the web app, you can find your password with this command:
+   ```shell
+   grep COUCHDB_PASSWORD /home/ubuntu/cht/upgrade-service/.env | cut -d'=' -f2
+   ```
+
+## TLS Certificates
+
+See the [TLS Certificates page]({{< relref "hosting/4.x/production/docker/adding-tls-certificates" >}}) for how to import your certificates.
+
+## Upgrades
+
+During upgrades, the CHT upgrade service updates the docker-compose files located in `/home/ubuntu/cht/compose/`. This means that any and all changes made to the docker-compose files will be overwritten. If there is ever a need to make any changes to the docker-compose files, be sure to re-do them post upgrades or should consider implementing them outside of those docker-compose files.
+
+### Upgrading the cht-upgrade-service
+
+The [CHT Upgrade Service](https://github.com/medic/cht-upgrade-service) provides an interface between the CHT Core API and Docker to allow easy startup and one-click upgrades from the CHT Admin UI. Occasionally, the CHT Upgrade Service, itself, will need to be upgraded. If an upgrade is available, it is highly recommended that you install the upgrade for the CHT Upgrade Service before performing further upgrades on your CHT instance. This is done via the following steps:
+
+1. Verify that the _version_ of the `cht-upgrade-service` image in your `./upgrade-service/docker-compose.yml` files is set to `latest`.
+1. Pull the latest `cht-upgrade-service` image from Docker Hub and replace the current container by running the following command:
+    ```shell
+    cd /home/ubuntu/cht/upgrade-service
+    docker compose pull
+    docker compose up --detach
+    ``` 
```

Comment:
Ditto

Reply:
These actually A) copy OK with the copy link or select and copy and B) are indented so they mach the numbered list indentation.

As such - OK to keep?